Update change log for 9.1.0-beta.2 (#295)
mapbox/mapbox-gl-native-android: CHANGELOG.md

## 9.1.0
This beta release includes a feature with a known issue: the `within` expression evaluates point features inconsistently across zoom levels if the point lies near the boundary of a GeoJSON object ([#16301](https://github.com/mapbox/mapbox-gl-native/issues/16301])).

> ### Snapshotter

Reviewer: Ah, sorry you moved faster than I could reply to your comment! I think we can delete this section and use the combined entry I suggested below, unless you have another suggestion @Chaoba

 - Added new API to add runtime layers/sources/images to MapSnapshotter . ([#268](https://github.com/mapbox/mapbox-gl-native-android/pull/268))
 - Added new API to get snapshotter's layers and sources and modify them by using the onDidFinishLoadingStyle() callback and APIs to retrieve layer/source properties from a style . ([#292](https://github.com/mapbox/mapbox-gl-native-android/pull/292))
 
### Features
 - Added `within` expression for testing whether an evaluated feature lies within a given GeoJSON object. `within` accepts `Feature`, `FeatureCollection`, `Polygon`, and `MultiPolygon` as valid GeoJSON objects. ([#198](https://github.com/mapbox/mapbox-gl-native-android/pull/198))

>  - Added `in` expression for testing whether an item exists in an array or a substring exists in a string. ([#171](https://github.com/mapbox/mapbox-gl-native-android/pull/171))

Reviewer: ```suggestion
 - Added `in` expression for testing whether an item exists in an array or a substring exists in a string. ([#171](https://github.com/mapbox/mapbox-gl-native-android/pull/171))
 - Introduced APIs and callbacks to customize the style used in a `MapSnapshotter` snapshotter with runtime styling methods. ([#268](https://www.github.com/mapbox/mapbox-gl-native-android/pull/268), [#292](https://www.github.com/mapbox/mapbox-gl-native-android/pull/292))
```

 - Added methods to set and get the sort key of features in a `FillLayer` and `LineLayer` at runtime. Features with a higher sort key will appear above features with a lower sort key. ([#209](https://github.com/mapbox/mapbox-gl-native-android/pull/209))
 - Introduced option to set the minimum and maximum pitch of a map. ([#199](https://github.com/mapbox/mapbox-gl-native-android/pull/199))
 - Introduced ability to configure the zoom level(s) that tiles are prefetched from by `Source`. This overrides the  tile prefetch setting defined by the `Map` instance. ([#184](https://github.com/mapbox/mapbox-gl-native-android/pull/184))

>  - Added new API to add runtime layers/sources/images to MapSnapshotter . ([#268](https://github.com/mapbox/mapbox-gl-native-android/pull/268))

Reviewer: This is a big feature and should be higher in the changelog! :) 

@tobrun @Chaoba @alexshalamov are there any other improvements to MapSnapshotter to call out besides the fact that you can now (1) manipulate the layers and sources of a style used by the snapshotter and (2) use runtime style property APIs on the snapshotter?

If so, I would be happy to call out the snapshotter in a separate section like @mapbox/maps-ios did: https://github.com/mapbox/mapbox-gl-native-ios/blob/master/platform/ios/CHANGELOG.md

If we made other improvements to the snapshotter that would make it more stable or thread-safe, etc., we should call that out, so let me know if there are additional details that we could include.

Author: @chloekraw afsk, there is no other more breaking changes. I will update the change log later to call out snapshotter in a separate section.

Reviewer: Ok! No problem. If there are no other changes, we should combine both entries into a single one and move the bullet under the new expressions we launched

 - Fixed an issue where a `LineDasharray` value of `[1, 0]` resulted in hairline gaps. ([#16202](https://github.com/mapbox/mapbox-gl-native/pull/16202))
 - Fixed Proguard configuration to prevent obfuscation of telemetry event classes. ([#195](https://github.com/mapbox/mapbox-gl-native-android/pull/195))
 - Fixed a crash that might occur during EGL window surface creation. ([#278](https://github.com/mapbox/mapbox-gl-native-android/pull/278))

>  - Fixed a bug that wrongly generated glyphs when high contrast accessibility mode was active by using RGB values instead of alpha channel. ([#289](https://github.com/mapbox/mapbox-gl-native-android/pull/289))

Reviewer: ```suggestion
 - Fixed a bug that would cause local CJK glyphs to be rendered incorrectly when generated on a device with the high-contrast accessibility setting enabled. ([#289](https://github.com/mapbox/mapbox-gl-native-android/pull/289))
```

### Other changes

>  - Updated mapbox-events-android to v4.7.3. ([#201](https://github.com/mapbox/mapbox-gl-native-android/pull/201))

Reviewer: ```suggestion
 - Updated mapbox-events-android to v5.0.0. ([#294](https://github.com/mapbox/mapbox-gl-native-android/pull/294))
```

>  - Revert "Transition JobIntentService to WorkManager in telemetry(#453).  ([#467](https://github.com/mapbox/mapbox-events-android/pull/467))

Reviewer: Since this change (this telemetry upgrade) was never released in a stable release, we do not need to mention the revert in changelog.md. In other words, changes between in pre-releases don't need to be in the changelog.md file.

>  - Revert "Transition JobIntentService to WorkManager in telemetry(#453).  ([#467](https://github.com/mapbox/mapbox-events-android/pull/467))

Reviewer: ```suggestion
```
